tests/scripts/pre-validation.sh

```diff
@@ -21,28 +22,45 @@ TF_VARS_FILE="terraform.tfvars"
     # $VALIDATION_APIKEY is available in the catalog runtime
     echo "ibmcloud_api_key=\"${VALIDATION_APIKEY}\""
     echo "region=\"${REGION}\""
-    echo "prefix=\"rag-da-$(openssl rand -hex 2)\""
+    echo "prefix=\"${PREFIX}\""
   } >> ${TF_VARS_FILE}
   terraform apply -input=false -auto-approve -var-file=${TF_VARS_FILE} || exit 1
 
+  prefix_var_name="prefix"
+  resource_group_name_var_name="resource_group_name"
   toolchain_region_var_name="toolchain_region"
+  toolchain_resource_group_var_name="toolchain_resource_group"
   ci_pipeline_id_var_name="ci_pipeline_id"
   cd_pipeline_id_var_name="cd_pipeline_id"
   watson_assistant_instance_id_var_name="watson_assistant_instance_id"
   watson_assistant_region_var_name="watson_assistant_region"
   watson_discovery_instance_id_var_name="watson_discovery_instance_id"
   watson_discovery_region_var_name="watson_discovery_region"
+  watson_machine_learning_instance_crn_var_name="watson_machine_learning_instance_crn"
+  watson_machine_learning_instance_guid_var_name="watson_machine_learning_instance_guid"
+  watson_machine_learning_instance_resource_name_var_name="watson_machine_learning_instance_resource_name"
 
+  resource_group_name_value=$(terraform output -state=terraform.tfstate -raw resource_group_name)
```

Comment:
you need to add all of these new outputs to the "tests/resources/existing-resources/outputs.tf"

Reply:
I think this is done now